game_api.txt

```diff
@@ -362,6 +362,7 @@ The farming API allows you to easily register plants and hoes.
 Fire API
 --------
 
+Add group flammable when registering a node to make fire seek for it.
```

Comment:
Group flammable can also be applied on items, maybe that should be documented here.
```suggestion
Add group flammable when registering a node to make fire seek for it.
Add it to an item to make it burn up when dropped in lava or fire.
```

Reply:
I think lava doesn't burn items by default, but there are mods for this.

Reply:
It does: https://github.com/minetest/minetest_game/blob/master/mods/default/item_entity.lua#L56

Reply:
You're right, sorry.